### ISSUE TITLE
Opening a past merge request of an unexistent branch should fail gracefully 

### ISSUE THREAD

**Issue by nunoalvesmsglife** [explore-dev/semantic-code-reviews-community#10]
**What happened**: Opened an old merge request and got an unexpected error

**What you expected to happen**: Either still be able to open it or a message explaining it can't be opened since the source branch no longer exists

**How to reproduce it (as minimally and precisely as possible)**: Create branch > merge request > accept and close origin branch > attempt to open merge request in ReviewPad

**Anything else we need to know?**:

```
{"latency":0.000117593,"message":"Finished gRPC request","request-id":"70037aa3-a4c4-42e6-b135-f089d62077c3","severity":"info","status":0,"time":"2021-01-19T09:41:41Z"}
{"latency":0.000373104,"message":"Finished gRPC request","request-id":"2a9afc0a-edcb-4464-bf24-5914399692d8","severity":"info","status":2,"time":"2021-01-19T09:41:41Z"}
{"message":"failed to get commits: rpc error: code = Unknown desc = errors in lookup headBranch: reference 'refs/merge-requests/23/head' not found ; cannot locate remote-tracking branch 'feature/4837'","request-id":"2a9afc0a-edcb-4464-bf24-5914399692d8","severity":"error","time":"2021-01-19T09:41:41Z"}

```

**Environment**:
- Extension version:
- Browser & version:
- Others: Gitlab 13.7

**Comment by marcelosousa** [comment]
@nunoalvesmsglife thank you for this feedback! 
We're aware of this issue - for some closed merge requests it is possible that the branch does not exist anymore. We'll be adding an info message for those code reviews.

**Comment by Xavier-explore-dev** [comment]
We need a better error message, but there may be a way of circumventing this problem for most PRs. I will check that.

**Comment by Xavier-explore-dev** [comment]
Addressed problem for most common case closed reviews without base branch: when base branch does not exist, we fallback to relying on hosts api to retrieve base and head commits.

**Comment by Xavier-explore-dev** [comment]
Fix deployed in version 0.21.
